syntax = "proto3";

package currencyconverter;

service CurrencyConverter {
  rpc ConvertCurrency(ConvertCurrencyRequest) returns (ConvertCurrencyResponse);
}

message ConvertCurrencyRequest {
  string from_currency = 1;
  string to_currency = 2;
  double amount = 3;
}

message ConvertCurrencyResponse {
  string message = 1;
}
